free-3.md

```diff
@@ -10,41 +10,72 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+A lot of people decorate their place with lights including lamps and string
+lights. Those lights are already manufactured and come in only certain
+styles, available at places like Ikea. I wish to change their experience by 
+making it easy for them to design their own lights so that they can submit 
+their own design to the manufacturers and get the customized lights.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Its domain is very specific to users who want to create their own design of 
+lights with ease. They would be provided with a lot of functions that let 
+them customize their lights, such as the number of bulbs, bulbs wrapped in 
+different shapes, mixing different shaped bulbs, length of the string, the 
+voltage, etc. 
```

Comment:
Awesome! I would love this. I just have the boring/classic globe lights in my room.